cloudmarker/clouds/azcloud.py

```diff
@@ -111,6 +114,9 @@ def read(self):
 
                     _get_record(resource_list, 'resource',
                                 subscription_id, self._max_recs),
+
+                    _get_record(mysql_server_list, 'my_sql_server',
+                                subscription_id, self._max_recs),
```

Comment:
Could you please rebase this pull request on the latest `master` branch? This part of the change seems to be based on an older `master` branch. A few merge conflicts have to be resolved after rebasing this on the latest `master`.

Reply:
Rebased the PR on latest `master` branch